styx-scheduler-service/src/test/java/com/spotify/styx/api/SchedulerResourceTest.java

```diff
@@ -103,6 +103,9 @@
 
   private final DockerImageValidator dockerImageValidator = mock(DockerImageValidator.class);
 
+  public SchedulerResourceTest() throws InterruptedException {
```

Comment:
why?

Reply:
Some IntelliJ suggestion that got forgotten in the process. Removed :)